### DISCUSSION TITLE
Searxng and MySQL database connection issues

### DISCUSSION THREAD

**Discussion by kingmoe001** [searxng/searxng#3128] [Q&A] [answered]
I am trying to use the MySQL engine on searxng to return results from my database. However, every time I run the webserver i get this error message:

```
* Serving Flask app 'webapp'
 * Debug mode: off
2024-01-16 16:35:39,637 ERROR:searx.webapp: 'title'
Traceback (most recent call last):
  File "/usr/local/searxng/searxng-src/searx/webapp.py", line 661, in search
    result_container = search.search()
  File "/usr/local/searxng/searxng-src/searx/search/__init__.py", line 212, in search
    self.result_container.close()
  File "/usr/local/searxng/searxng-src/searx/results.py", line 360, in close
    result['title'] = ' '.join(utils.html_to_text(result['title']).strip().split())
KeyError: 'title'
```

I believe the issue to be related to the MySQL engine because when I turn off the feature in searxng preferences, I can retrieve search results and this error message no longer pops up. 
![Screenshot (7)](https://github.com/searxng/searxng/assets/48866202/ea0d3dcd-16e0-4049-95d9-17b4b5886c48)

Here is my settings.yml file server and formats configuration. 
![Screenshot (8)](https://github.com/searxng/searxng/assets/48866202/a05aff99-569b-4db8-8cd7-6167d528966f)

and MySQL 
![Screenshot (9)](https://github.com/searxng/searxng/assets/48866202/2f82cc12-fc49-4ab8-b559-f54a635ab325)

Also, I have checked - the rate limiter plugin is disabled

Any suggestions?

**Comment by return42** [comment, marked as answer]
> Any suggestions?

Its a bug in SearXNG .. I opened https://github.com/searxng/searxng/issues/3130
